README.md

```diff
@@ -1,11 +1,13 @@
-# load-rhs-notebook-python
+# Intan RHS Single File Data Loader
 
-To run this example, either download the 3 files:
-1. LoadIntanRHS_Python.ipynb
-2. importintanutilities.py
-3. sampledata.rhs
+`importrhsutilities.py` is renamed to `intanrhsreader.py`.
```

Comment:
Thanks.  Were any other changes made to the file?  Black formatting? 
 The files are off by 98 lines.  I want to document the changes now so that it is easier to pull from upstream in the future.

Reply:
I documented all the changes in the CHANGELOG.md file.